src/cmd/run.js

```diff
@@ -296,7 +297,8 @@ export default async function run(
     log.debug('Not installing as temporary add-on because the ' +
               'add-on was already installed');
   } else if (requiresRemote) {
-    client = await firefoxClient();
+    const port = runningFirefox.spawnargs[2];
```

Comment:
We already started firefox at this point, `runningFirefox` is the process returned from `fx-runner`. That we need here is retrieve the port in which the debugger server was started.

I was thinking in maybe extend the process. But I don't think that will be easy. So maybe we just can move when [find the port](https://github.com/mozilla/web-ext/blob/master/src/firefox/index.js#L136) and run that before [initiate the runner](https://github.com/ccarruitero/web-ext/blob/4f4b9efd5834f2603c8bb8c4d9d399652567b206/src/cmd/run.js#L271) and then pass it the to the runner and the client.